src/components/organisms/MessagesContainer.vue

```diff
@@ -1,22 +1,27 @@
 <template>
   <div class="flex">
-    <div class="w-full md:w-1/4 md:border md:border-r-0 md:border-gray">Inbox</div>
+    <div class="w-full md:w-1/3 md:border md:border-r-0 md:border-gray">
+      <inbox />
```

Comment:
We could make the inbox scrollable if there are more inbox items given. (overflow-auto or overflow-y-auto)